R/sparklyr-mosaic/tests.R

```diff
@@ -9,7 +9,8 @@ if(length(getOption("repos")) < 1) {
 install.packages("sparklyr", repos="")
 library(sparklyr)
 
-spark_home_set("/usr/spark-download/unzipped/spark-3.3.2-bin-hadoop3")
+spark_home <- Sys.getenv("SPARK_HOME")
+spark_home_set(spark_home)
 install.packages("sparklyrMosaic_0.3.14.tar.gz", repos = NULL)
```

Comment:
@milos-colic will "sparklyrMosaic_0.3.14.tar.gz" be what is used in Mosaic v0.4?